ownership.md

```diff
@@ -0,0 +1,23 @@
+# Team ownership
+
+This document details which areas are covered by each team member.
+Besides overall transparency, this might come in handy in case of vacations or sickness.
+
+|                                                  | [Jacqueline](https://github.com/jacqueline-kraus) | [Mostafa](https://github.com/mostafa-anm) | [Philipp](https://github.com/creativephil) | [Rustem](https://github.com/amureki) | [Sebastian](https://github.com/SebastianKapunkt) |
```

Comment:
```suggestion
|                                                  | [Jacqueline](https://github.com/jacqueline-kraus) | [Mos](https://github.com/mostafa-anm) | [Philipp](https://github.com/creativephil) | [Rustem](https://github.com/amureki) | [Sebastian](https://github.com/SebastianKapunkt) |
```

I would use simple @-handles here. I would also go by order of joining date, not a-z. Makes it simpler to add people.